src/lxc/attach.c

```diff
@@ -932,7 +931,7 @@ int lxc_attach(const char* name, const char* lxcpath, lxc_attach_exec_t exec_fun
 	}
 
 	if (options->attach_flags & LXC_ATTACH_MOVE_TO_CGROUP && cgns_supported())
```

Comment:
This is just style preference but I'd like this to have extra brackets so that the intentions are clear: 
`if ((options->attach_flags & LXC_ATTACH_MOVE_TO_CGROUP) && cgns_supported())`